esx_service/cli/vmdkops_admin_test.py

```diff
@@ -677,6 +692,7 @@ def test_tenant_vm(self):
         # create tenant1 without adding any vms and privilege
         error_info, tenant = auth_api._tenant_create(
                                                     name=self.tenant1_name,
+                                                    default_datastore=auth_data_const.VM_DS,
```

Comment:
Does the tenant_vm_ls() show the actual datastores for each VM or VM_DS? Is VM_DS preferrable vs. showing different datastores per VM?